raiden/ui/web/src/assets/config/config.development.json

```diff
@@ -2,5 +2,6 @@
   "raiden": "/api/1",
   "reveal_timeout": 10,
   "settle_timeout": 500,
-  "web3": "/web3"
+  "web3": "/web3",
+  "network_type": 2
```

Comment:
As talked with @LefterisJP , `network_type` should be a lowercase string, either `main` or `test`

Reply:
I will update the PR shortly with the changes :)